server/test/commandHandlers/deployment/ProvideInfrastructureTest.js

```diff
@@ -823,22 +823,22 @@ describe('ProvideInfrastructureCommandHandler:', () => {
       // Mocking AutoScalingTemplatesProvider
       let expectedAutoScalingBlueTemplate = {
         autoScalingGroupName: 'pr1-ta-Web-blue',
-        launchConfigurationName: 'LaunchConfig_pr1-ta-Web-blue'
+        image: { rootVolumeSize: 10 }, devices: [{ DeviceName: '/dev/sda1', Ebs: { VolumeSize: 20 }}], launchConfigurationName: 'LaunchConfig_pr1-ta-Web-blue'
       };
 
       let expectedAutoScalingGreenTemplate = {
         autoScalingGroupName: 'pr1-ta-Web-green',
-        launchConfigurationName: 'LaunchConfig_pr1-ta-Web-green'
+        image: { rootVolumeSize: 10 }, devices: [{ DeviceName: '/dev/sda1', Ebs: { VolumeSize: 20 }}], launchConfigurationName: 'LaunchConfig_pr1-ta-Web-green'
       };
 
 
       // Mocking LaunchConfigurationTemplatesProvider
       let expectedLaunchConfigurationBlueTemplate = {
-        launchConfigurationName: 'LaunchConfig_pr1-ta-Web-blue'
+        image: { rootVolumeSize: 10 }, devices: [{ DeviceName: '/dev/sda1', Ebs: { VolumeSize: 20 }}], launchConfigurationName: 'LaunchConfig_pr1-ta-Web-blue'
       };
 
       let expectedLaunchConfigurationGreenTemplate = {
-        launchConfigurationName: 'LaunchConfig_pr1-ta-Web-green'
+        image: { rootVolumeSize: 10 }, devices: [{ DeviceName: '/dev/sda1', Ebs: { VolumeSize: 20 }}], launchConfigurationName: 'LaunchConfig_pr1-ta-Web-green'
```

Comment:
Nothing major: I like to create helper functions to create these expected data sets. It makes changing it easier, it makes reading the tests simpler.